pipestat/cli.py

```diff
@@ -35,23 +35,22 @@ def main():
         status_schema_path=args.status_schema,
         flag_file_dir=args.flag_dir,
     )
+    types_to_read_from_json = ["object"] + list(CANONICAL_TYPES.keys())
     if args.command == REPORT_CMD:
         value = args.value
         if psm.schema is None:
             raise SchemaNotFoundError(msg="report", cli=True)
         result_metadata = psm.schema[args.result_identifier]
```

Comment:
Now returning ParsedSchema instead of schema, should be:
`result_metadata = psm.schema.results_data[args.result_identifier]`